.vscode/launch.json

```diff
@@ -5,14 +5,11 @@
     "version": "0.2.0",
     "configurations": [
         {
-            "name": "main.py ",
+            "name": "Python Debugger: Current File",
             "type": "debugpy",
             "request": "launch",
-            "program": "${workspaceFolder}/core/main.py",
-            "console": "integratedTerminal",
-            "args": [
-                "test.skip_transform=True"
-            ]
+            "program": "${file}",
+            "console": "integratedTerminal"
         }
     ]
 }
```

Comment:
Don't commit this. We're not officially supporting any debugging methods. The file probably erroneously snuck in a previous commit. We should add it to the gitignore for now.